extensions/terminal-suggest/src/terminalSuggestMain.ts

```diff
@@ -493,13 +493,9 @@ export async function getCompletionItemsFromSpecs(
 		}
 		showFiles = true;
 		showFolders = true;
-	}
-	// For arguments when no fig suggestions are found these are fallback suggestions
-	else if (!items.length && !showFiles && !showFolders && !hasCurrentArg) {
-		if (terminalContext.allowFallbackCompletions) {
-			showFiles = true;
-			showFolders = true;
-		}
+	} else if (!items.length && !showFiles && !showFolders && !hasCurrentArg) {
```

Comment:
@Tyriar I think we still want this, correct?

Reply:
As in the `else if` case? It should depend upon `unknown = 'on'` now I think since that's no longer coming through as `wasExplicitlyInvoked`